xs2a-adapter-service-impl/src/test/java/de/adorsys/xs2a/adapter/impl/http/wiremock/WiremockFileResolverTest.java

```diff
@@ -0,0 +1,373 @@
+    private static final String PERIODIC_SCT_URI = "/v1/periodic-payments/sepa-credit-transfers";
+    private static final String PERIODIC_PAIN001_CST_URI = "/v1/periodic-payments/pain.001-sepa-credit-transfers";
+    // Flowed data
+    private static final String FLAWED_URI = "/foo/boo";
+    private static final String FLAWED_BODY = "{\"foo\": \"boo\"}";
+    private static final String FLAWED_METHOD = "FOO";
+    // Decent data
+    private static final String BOO_AUTHORISATIONS_URI = "/boo/authorisations";
+    private static final String BOO_STATUS_URI = "/boo/status";
+    // AIS
+    private static final String CONSENT_STATUS_URI = CONSENTS_URI + BOO_STATUS_URI;
+    private static final String AIS_START_PSU_AUTHORISATION_URI = CONSENTS_URI + BOO_AUTHORISATIONS_URI;
+    private static final String AIS_SCA_URI = AIS_START_PSU_AUTHORISATION_URI + "/foo";
+    private static final String TRANSACTIONS_URI = ACCOUNTS_URI + "/boo/transactions";
+    // PIS
+    private static final String PAYMENTS_SCT_AUTHORISATIONS_URI = PAYMENTS_SCT_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PAYMENTS_PAIN001_SCT_AUTHORISATIONS_URI = PAYMENTS_PAIN001_SCT_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PERIODIC_SCT_AUTHORISATIONS_URI = PERIODIC_SCT_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PERIODIC_PAIN001_SCT_AUTHORISATIONS_URI = PERIODIC_PAIN001_CST_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PAYMENTS_SCT_PAYMENT_STATUS_URI = PAYMENTS_SCT_URI + BOO_STATUS_URI;
+    private static final String PAYMENTS_PAIN001_SCT_PAYMENT_STATUS_URI = PAYMENTS_PAIN001_SCT_URI + BOO_STATUS_URI;
+    private static final String PERIODIC_SCT_PAYMENT_STATUS_URI = PERIODIC_SCT_URI + BOO_STATUS_URI;
+    private static final String PERIODIC_PAIN001_SCT_PAYMENT_STATUS_URI = PERIODIC_PAIN001_CST_URI + BOO_STATUS_URI;
+    private static final String PAYMENTS_SCT_SCA_STATUS_URI = PAYMENTS_SCT_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PAYMENTS_PAIN001_SCT_SCA_STATUS_URI = PAYMENTS_PAIN001_SCT_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PERIODIC_SCT_SCA_STATUS_URI = PERIODIC_SCT_URI + BOO_AUTHORISATIONS_URI;
+    private static final String PERIODIC_PAIN001_SCT_SCA_STATUS_URI = PERIODIC_PAIN001_CST_URI + BOO_AUTHORISATIONS_URI;
+    // Both
+    private static final String PSU_DATA_BODY = "{\"psuData\": \"\"}";
+    private static final String AUTHENTICATION_METHOD_ID_BODY = "{\"authenticationMethodId\": \"\"}";
+    private static final String SCA_AUTHENTICATION_DATA_BODY = "{\"scaAuthenticationData\": \"\"}";
+
+    @Test
+    void aisCreateConsent() {
+        assertTrue(AIS_CREATE_CONSENT.check(CONSENTS_URI, POST_METHOD, null));
+
+        assertFalse(AIS_CREATE_CONSENT.check(FLAWED_URI, POST_METHOD, null));
+        assertFalse(AIS_CREATE_CONSENT.check(CONSENTS_URI, FLAWED_METHOD, null));
+    }
+
+    // todo: add start psu authentication
```

Comment:
can add more details into todo or link the issue. for me, it is not clear

Reply:
My bad, it was the reminder for myself, I should've removed it before pr. 
Erased